onadata/apps/logger/models/instance.py

```diff
@@ -335,10 +335,9 @@ def get_full_dict(self, load_existing=True):
             if isinstance(self.deleted_at, datetime):
                 doc[DELETEDAT] = self.deleted_at.strftime(MONGO_STRFTIME)
 
-            if self.get_review_status():
+            # pylint: disable=E1101
```

Comment:
Please write the English name of the rule you are disabling (for readability) is there is space on the line